languages/clojure/exercises/concept/basic-functions/.doc/instructions.md

```diff
@@ -0,0 +1,15 @@
+In this exercise you'll be creating some functions which work with the number 42, sometimes referred to as "the answer to the ultimate question of life, the universe."
+
+You have three tasks.
+
+### 1. Create a function that returns 42
+
+First, create a function named `new-function` that takes no arguments and returns the number 42 every time it is invoked.
+
+### 2. Create a function that accepts an input and adds 42 to it 
+
+Now create a function named `plus-42` that takes one input, `x`, and returns the sum of that argument and the number 42.
+
+### 3. Create a function that takes and argument and calls function defined in (2) with it
+
+Finally, create a function named `invoke-plus-42` that accepts an input,`y`, uses it as the argument with which to call the `plus-42` function, and then returns the result of that function invocation.
```

Comment:
I would add 
```suggestion
Finally, create a function named `invoke-plus-42` that accepts an input,`y`, uses it as the argument with which to call the `plus-42` function, and then returns the result of that function invocation.

### 4. Create a predicate function named `is-multiple-of-42?`  that takes an input, `x`, and returns true if x is divisible by 42.
```

Reply:
I like the idea to add a predicate function, but I feel that it's testing them on multiple things with the divisibility check. What do you think about just `is-arg-42?` which evaluates to either true or false depending on whether the argument passed in is the number 42?

Reply:
If we are adding this we may want to mention functions ending in `!` as well and any other idiomatic Clojure function naming conventions.  👍

Reply:
Usually in Clojure we don't use `is` when naming predicates. 'multiple-of-42?` would be a little more idiomatic.

Introducing the `!` convention as this stage might be a little much before we introduce the concept of pure functions and side effects?

Reply:
I would prefer mentioning more idiomatic Clojure func naming conventions, no need to put all into exercises, but just mention in the instruction and for details let learners google. (I don't know of the `!` convention if you didn't mention it above~)

Reply:
I like referring to a document listing common naming conventions. This is a perfect case for the `after.md` document. You don't need to include it in the `instructions.md` document as we're being explicit about the function names here anyway.